javascript/gdx_getting_started_3.html

```diff
@@ -0,0 +1,96 @@
+<!--
+This example uses the GDX JavaScript library to collect and manipulate data from a Vernier Go Direct® Sensor.
+Press the 'Select Go Direct Device' button to open communication to the device in a Chrome browser.
+Press the 'Change Sensor and Collect Data' to change the sensor channel and collect 10 data points. 
+
+This program uses the following functions from the GDX Library:
+
+gdxDevice.selectDevice();
+gdxDevice.close();
+gdxDevice.sensors.filter();
+gdxDevice.getSensor(int);
+gdxDevice.start();
+gdxDevice.stop();
+sensor.setEnabled(bool);
+-->
+
+<!doctype html>
+<html>
+<head>
+  <title>Example 3: User Prompts and Collecting Data</title>
+  <meta name="description" content="The gdx_getting_started_3.html example shows you how to use the library:
+Connect to a Go Direct device over BLE; Provide the user with a prompt to select the active sensors; Provide the user with a prompt to set the measurement period; Collect a few measurements from the enabled sensors; and
+Disconnect">
+  <meta charset="utf-8">
+  <script src="https://unpkg.com/@vernier/godirect/dist/godirect.min.umd.js"></script>
+</head>
+
+<body>
+  <button id="select_device">Select Go Direct Device</button>
+  <button id ="start_collection" onclick = "chooseSensor()">Change Sensor and Collect Data</button>
```

Comment:
Suggest changing button label to read, "Select Sensor and Collect Data". 
Clarifies what user will do.

Reply:
done